lib/migrator.js

```diff
@@ -148,13 +162,31 @@ module.exports = async function(args) {
 
       if (title.includes('"')) title = title.replace(/"/g, '\\"');
 
+      const newPostCats = [];
+      const filterPostCats = postCats => {
+        postCats.forEach(cat => {
+          const siteCat = siteCatsObj[cat] ? siteCatsObj[cat] : '';
+          const newPostCatsObj = arrayToObj(newPostCats)[cat];
+
+          // Avoid duplicate objects
+          if (typeof newPostCatsObj === 'undefined') newPostCats.push({ name: cat, parent: siteCat });
+
+          if (siteCat) {
+            filterPostCats([siteCat]);
```

Comment:
Most probably this is not needed because WP should include all ancestors of a category.

@adnan360 

Let say the site categories are:
1. lorem => ipsum => dolor
2. foo => bar
3. baz

If I add "ipsum" and "bar" categories to a post, WP automatically includes "lorem" and "foo" as well, right?

Reply:
No, I think not. I tried with a new post "Test Post" and added a subcategory to it:
![wp-temp-01](https://user-images.githubusercontent.com/7857310/88771451-188c2900-d1a1-11ea-8604-83f35c943ae7.png)
[Export](https://gist.github.com/adnan360/58af6a0b1144ea51a262a0f0a6882338) has only the subcategory:
```
...
		<wp:is_sticky>0</wp:is_sticky>
										<category domain="category" nicename="test-subcategory-2"><![CDATA[Test Subcategory 2]]></category>
						</item>
```

Reply:
I see, my initial worry was warranted then.